haas/ext/obm/ipmi.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2013-2014 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""IPMI driver for implementing out of band management. """
+
+from sqlalchemy import Column, String, Integer, ForeignKey
+import schema
+
+
+from haas.model import Obm
+from haas.errors import OBMError
+from haas.dev_support import no_dry_run
+
+class Ipmi(Obm):
```

Comment:
Putting it here to avoid repetition of what was discussed between I, George and Jonathan.

We have decided not to put the name IpmiObm and continue with Ipmi. Only in case of HaaS acting as OBM or Switch or some other role on behalf of recursive HaaS we will be using the name HaaS<driverTypeName> and maintain the camel case for the classes.